examples/basic/database_sql/data.go

```diff
@@ -48,7 +48,12 @@ func episodeData(seriesID, seasonID, episodeID, title string, date time.Time) ty
 	)
 }
 
-func getData() (series, seasons, episodes []types.Value) {
+func getData() ([]types.Value, []types.Value, []types.Value) {
```

Comment:
Read comment for getDataForITCrowd

Reply:
OK. I returned previous declaration.
I simple followed by linter's comment.